requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Include everything the framework requires
 # You will automatically get updates for all versions starting with "1.".
 rlbot==1.*
+rlbottraining
```

Comment:
The new GUI is capable of downloading the source code of PythonExampleBot to get people started quickly. Currently it does not install requirements.txt after the download, but maybe it should. I can take care of that, but I wanted to get your recommendation.

Reply:
I think whenever we import any python bot, we should look for requirements.txt to install.

Reply:
OK, I'll look into that.